docs/product-ibm-websphere-installation-and-configuration-guide/src/main/asciidoc/configure-was-proc.adoc

```diff
@@ -0,0 +1,341 @@
+[id='configure-server']
+= Configuring WAS for {KIE_SERVER}
+
+Before you can deploy {KIE_SERVER} on WAS, you must configure the server to accept the deployable `kie-execution-server.war` file.
+
+.Prerequisites
+You have successfully logged in to the IBM WebSphere's *IBM Integrated Solutions Console*. The main menu on the left side of the console contains all the links necessary for configuring the application server.
+
+.IBM Integrated Solutions Console
+image::websphere_console.png[]
+
+== Increasing JVM heap size
+
+The default JVM heap size in WAS may cause errors when deploying {KIE_SERVER}. To avoid issues, increase the heap size:
+
+. In the *Integrated Solutions Console*, go to *Servers* -> *Server Types* -> *WebSphere application servers*.
+. From the list of application servers, click on the server that you are going to deploy {KIE_SERVER}. For example, `server1`.
++
+The configuration page for `server1` opens.
+. Under *Server Infrastructure*, expand *Java and Process Management* and click *Process definition*.
++
+.Application server configuration page
+image::process_definition.png[]
+. Click *Java Virtual Machine* under *Additional Properties*.
++
+.Process definition configuration page
+image::process_definition2.png[]
++
+The *General Properties* for the JVM that is used to start the server page opens.
+. Change both the *Initial heap size* and *Maximum heap size* to `2048`. {KIE_SERVER} has been tested with this value.
++
+.JVM configuration page
+image::process_definition3.png[]
+. Click *Apply*.
+. Click *Save* in the *Messages* window to save your changes to the master configuration.
++
+.Messages pop-up
+image::messages_popup.png[]
+. Stop and restart the server in your terminal by navigating to the WebSphere Application Server `/bin` directory location that you specified during installation.
+. Run the following commands:
++
+
+[source]
+----
+sudo ./stopServer.sh server1
+----
++
+
+[source]
+----
+sudo ./startServer.sh server1
+----
+
+[id='_modify_security_settings']
+== Modifying Security Settings
+
+Modify security settings on IBM WebSphere Application Server to enable the container-managed authentication mechanisms provided by the server.
+
+. In the main menu, click *Security* -> *Global Security*. Ensure that the option *Enable Application Security* is checked. This may already be checked and overridden at the server level.
++
+.Global Security Configuration Page
+image::global_security.png[]
+
+. Click *Custom Properties* on the right side and then *New...* to enter a new custom property with the following details:
```

Comment:
You can delete this custom property configuration, it isn't needed.

Reply:
Done